Volume/Views/SettingsView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SettingsView.swift
+//  Volume
+//
+//  Created by Cameron Russell on 4/12/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct SettingsView: View {
+    static let pages = [
+        SettingsPage(destination: .externalLink(Secrets.feedbackForm), imageName: "flag", info: "Send Feedback"),
+        SettingsPage(destination: .externalLink("https://www.cornellappdev.com/"), imageName: "link", info: "Visit Our Website"),
+        SettingsPage(destination: .internalView(.aboutUs), imageName: "info", info: "About Us"),
+    ]
+    
+    init() {
+        UINavigationBar.appearance().titleTextAttributes = [.font : UIFont(name: "Begum-Medium", size: 20)!]
```

Comment:
Didn't we define something like `Font.begumMedium` in extensions?

Reply:
Font is a View whereas .font is a UIFont, and there's no way to cast between the two